.changeset/wicked-spies-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@qiskit/web-components': minor
```

Comment:
This looks good to me

@eddybrando - are you good w/ this PR?

Reply:
Yes, it looks good now 🙂